jobs/build/sync-for-ci/Jenkinsfile

```diff
@@ -127,15 +131,28 @@ node {
 
                 stage("push to s3") {
                     commonlib.syncRepoToS3Mirror("${LOCAL_SYNC_DIR}/", "${S3_SYNC_DIR}/")
+                    if ( commonlib.ocp4Versions.get(0) == REPOSYNC_DIR ) {
+                        // create 4.(x+1) dir to meet CI when branch cut
+                        def S3_SYNC_DIR_NEXT = S3_SYNC_DIR.replaceAll("${REPOSYNC_DIR}","${MAJOR}.${MINOR+1}")
```

Comment:
String replacement doesn't seem to be appropriate here because you are assuming `REPOSYNC_DIR` is not repeating. I would suggest concatenating the desired path rather than replacing a substring in the original path.